patches/chromium/expose_a_method_for_invalidating_internal_cache.patch

```diff
@@ -0,0 +1,43 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "emmanuel.kimmerlin@thomsonreuters.com"
+ <emmanuel.kimmerlin@thomsonreuters.com>
+Date: Mon, 18 Nov 2019 12:42:23 +0100
+Subject: Expose a method for invalidating internal cache
+
+Allow to customize Electron/Chromium cache location
+See MR: https://github.com/electron/electron/pull/19693
```

Comment:
Per https://www.electronjs.org/docs/development/patches#patch-justification, every patch we carry is a significant ongoing maintenance burden. We ask that the description of a patch includes a justification of why they're worth that maintenance burden, in addition to a description of the functionality that the patch adds.

In order of descending desirability:

1. Can this patch be removed entirely by achieving the desired functionality without having to include a patch?
2. Can this patch be merged upstream into Chromium itself? (see [here](https://chromium.googlesource.com/chromium/src/+/master/docs/contributing.md) for a guide on contributing to Chromium)
3. In the patch description, describe the reasons why neither of the above two are possible, and include guidance for a future maintainer looking to clean up patches about how they can check whether this patch is still required (maybe as simple as "if it compiles without this patch, the patch is no longer needed").

Thanks for helping us keep maintaining Electron sustainable 🙇